ui/package.json

```diff
@@ -3,16 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@agoric/assert": "*",
```

Comment:
Did you define these values?

Reply:
was having issue with this version of agoric assert thats why I changed it but now it working fine.

Reply:
Please revert to original versions and try to run it. Let's not change this unless required.